src/main/java/edu/harvard/iq/dataverse/license/dto/LicenseMapper.java

```diff
@@ -36,6 +38,27 @@ public LicenseDto mapToDto(License license) {
         return licenses.stream().map(this::mapToDto).collect(Collectors.toList());
     }
 
+    public LicenseSimpleDto mapToSimpleDto(License license, Locale textLocale) {
+
+        for (LocaleText localeLicenseText : license.getLocalizedNames()) {
+            if (localeLicenseText.getLocale().equals(textLocale)) {
+                return new LicenseSimpleDto(license.getId(), localeLicenseText.getText());
+            }
+        }
```

Comment:
You can do `return new LicenseSimpleDto(license.getId(), license.getLocalizedName(textLocale));` instead of this loop